spec/controllers/training_modules_users_controller_spec.rb

```diff
@@ -30,6 +33,17 @@
           .to be_between(1.minute.ago, 1.minute.from_now)
       end
     end
-  end
+    context 'nonexistent slide' do
+      let(:nonexistent_slide_params) do
+        { module_id: t_mod.slug,
+          slide_id: 'not-a-real-slide',
+          user_id: user.id }
+      end
 
+      it 'renders a response' do
+        post :create_or_update, nonexistent_slide_params, format: :json
+        expect(response).to be
```

Comment:
(I assume you've seen [this](http://qz.com/283268/if-shakespeare-wrote-in-javascript-heres-what-it-would-look-like/).)

Anyway. Maybe check the response status?